cmd/snap-update-ns/change.go

```diff
@@ -499,9 +499,19 @@ func neededChanges(currentProfile, desiredProfile *osutil.MountProfile) []*Chang
 		desired[i].Dir = filepath.Clean(desired[i].Dir)
 	}
 
+	logger.Debugf("current profile:\n%v", current)
+	logger.Debugf("desired profile")
+	for _, en := range desired {
+		logger.Debugf("- %v", en)
+	}
```

Comment:
we should have a small helper for this, to make the new debug code a bit less distracting